### PR TITLE
Multi-terms aggregations docs

### PR DESCRIPTION
**Pull request by alicejw-aws**
…/issues/582

Signed-off-by: alicejw <alicejw@amazon.com>

### Description
New feature docs for multi-term aggregation. 

### Issues Resolved
Closes [582](https://github.com/opensearch-project/documentation-website/issues/582)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/bucket-agg.md [hunk 1]
```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
 
 This is especially true if `size` is set to a low number. Because the default size is 10, an error is unlikely to happen. If you don’t need high accuracy and want to increase the performance, you can reduce the size.
 
+## Multi-terms
+
+Similar to the `terms` bucket aggregation, you can also search for multiple terms using the `multi_terms` aggregation.
+
+You can sort by a number of documents. You can also sort by a metric aggregation on a composite key and get the top `n` results. Consider create a search that contains several terms: the number of documents (e.g. 1000), and the number of servers per location that show CPU usage greater than 90%.
```
Comment (natebower) [inline _opensearch/bucket-agg.md:81]: Third sentence: Should this be "Consider creating..."? Also, we say "several," but it looks like we only provide two terms. Insert a comma after "e.g." and in "1,000" (e.g., 1,000). Remove the comma after the parenthetical.

File: _opensearch/bucket-agg.md [hunk 2]
```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
...
+This multi-term query returns the top number of results.
+
+The multi_terms aggregation does consume more memory, so performance will be slower than a `terms` aggregation.
```
Comment (natebower) [inline _opensearch/bucket-agg.md:84]: I suggest changing this sentence to "The multi_terms aggregation does consume more memory than a 'terms' aggregation, so its performance will be slower."

File: _opensearch/bucket-agg.md [hunk 3]
```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
...
+{: .tip }
+
+### Multi-term parameters
+
+Parameter | Description
+:--- | :---
+size | Specifies the amount of buckets to return. Default is 10.
```
Comment (natebower) [inline _opensearch/bucket-agg.md:91]: Change "amount" to "number".

File: _opensearch/bucket-agg.md [hunk 4]
```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
...
+order | To indicate the order of buckets. The default order is the number of buckets.
```
Comment (natebower) [inline _opensearch/bucket-agg.md:92]: Can this be "Indicates the order of buckets" in order to match the structure of the other two?

File: _opensearch/bucket-agg.md [hunk 5]
```diff
@@ -74,6 +74,123 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
...
+doc_count | Specifies the number of documents in each bucket for it to be returned. By default, the top ten terms are returned.
```
Comment (natebower) [inline _opensearch/bucket-agg.md:93]: I'm not following the wording of the first sentence in the description. Should it be something like "Specifies the number of documents to be returned in each bucket"? In the second sentence, change "ten" to "10".

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS Please see my comments and let me know if you have any questions. Thanks!

File: _opensearch/bucket-agg.md [hunk 6]
```diff
@@ -74,6 +74,121 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
 
 This is especially true if `size` is set to a low number. Because the default size is 10, an error is unlikely to happen. If you don’t need high accuracy and want to increase the performance, you can reduce the size.
 
+## Multi-terms
+
+Similar to the `terms` bucket aggregation, you can also search for multiple terms using the `multi_terms` aggregation. Multi-terms aggregations are useful when you need to sort by a number of documents, or when you need to sort by a metric aggregation on a composite key and get the top `n` results. For example, you could search for a specific number of documents (e.g., 1000) and the number of servers per location that show CPU usage greater than 90%. The top number of results would be returned for this multi-term query.
+
```
Comment (kolchfa-aws) [inline _opensearch/bucket-agg.md:80]: I don't quite understand what's meant by "sort by a number of documents". Maybe "sort by document count"?

**Review by kolchfa-aws — APPROVED** [review]
LGTM except for my comment.

File: _opensearch/bucket-agg.md [hunk 7]
```diff
@@ -74,6 +74,121 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
 
 This is especially true if `size` is set to a low number. Because the default size is 10, an error is unlikely to happen. If you don’t need high accuracy and want to increase the performance, you can reduce the size.
 
+## Multi-terms
+
+Similar to the `terms` bucket aggregation, you can also search for multiple terms using the `multi_terms` aggregation. Multi-terms aggregations are useful when you need to sort by document count, or when you need to sort by a metric aggregation on a composite key and get the top `n` results. For example, you could search for a specific number of documents (e.g., 1000) and the number of servers per location that show CPU usage greater than 90%. The top number of results would be returned for this multi-term query.
+
+The `multi_terms` aggregation does consume more memory than a `terms` aggregation, so its performance will be slower.
```
Comment (hdhalter) [inline _opensearch/bucket-agg.md:81]: I would exchange 'will' for 'may' as we don't want to promise them slower performance. : )

File: _opensearch/bucket-agg.md [hunk 8]
```diff
@@ -74,6 +74,121 @@ The `terms` aggregation requests each shard for its top 3 unique terms. The coor
...
+{: .tip }
+
+### Multi-terms aggregation parameters
+
+Parameter | Description
+:--- | :---
+multi_terms | Indicates a multi-terms aggregation that gathers buckets of documents together based on criteria specified by multiple terms.
+size | Specifies the number of buckets to return. Default is 10.
+order | Indicates the order of buckets. The default order is the number of buckets.
```
Comment (hdhalter) [inline _opensearch/bucket-agg.md:90]: This might need a little more context... how is the list ordered?

**Review by hdhalter — APPROVED** [review]
Looks good! Just a couple of comments.

**Pull request merged**